public-keys-component.js

```diff
@@ -0,0 +1,20 @@
+/*!
+ * Copyright (c) 2016 Digital Bazaar, Inc. All rights reserved.
+ */
+define(['angular'], function(angular) {
+
+'use strict';
+
+function register(module) {
+  module.component('brPublicKeys', {
```

Comment:
It turns out that `br-keys` actually does have the capability to be just a view of an identity's keys, but there was a small bug that broke that functionality (it would do a permission check to see if the user was able to edit a key, and if not it wouldn't display any edit functionality, but the permission function was accidentally removed a bit ago). Went ahead and removed `brPublicKeys` because it's redundant now.